chpl/chpl-service/src/main/java/gov/healthit/chpl/util/CertificationResultRules.java

```diff
@@ -165,6 +165,13 @@ private void setRulesUsingDatabase(CertificationCriterionAttributeDAO certificat
                         .optionName(TEST_TOOLS_USED)
                         .build());
             }
+            if (attribute.getTestData()) {
```

Comment:
I know it's a tiny thing, but the other rules in this method are sorted alphabetically. Can we move this one two rules up?